modules/RTC/RTCUtils.js

```diff
@@ -888,10 +833,7 @@ class RTCUtils extends Listenable {
                         navigator.mediaDevices.getUserMedia.bind(
                             navigator.mediaDevices),
                             true);
-                this.enumerateDevices
-                    = wrapEnumerateDevices(
-                        navigator.mediaDevices.enumerateDevices.bind(
-                            navigator.mediaDevices));
+                this.enumerateDevices = rawEnumerateDevicesWithCallback;
```

Comment:
It can be undefined but that's not what this PR is about. It was the same before, unless I'm missing something ?